fails build without `-Wno-error=unused-result`

builds fail if not set `CXXFLAGS=" -Wno-error=unused-result"`

~~~make
[ 66%] Building CXX object CMakeFiles/VulkanHppGenerator.dir/tinyxml2/tinyxml2.cpp.o
/tmp/makepkg/sl1-vulkan-git/src/Vulkan-Hpp/VulkanHppGenerator.cpp: En la función ‘int main(int, char**)’:
/tmp/makepkg/sl1-vulkan-git/src/Vulkan-Hpp/VulkanHppGenerator.cpp:16519:16: error: ignoring return value of ‘int system(const char*)’ declared with attribute ‘warn_unused_result’ [-Werror=unused-result]
16519 |     std::system( commandString.c_str() );
      |     ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
cc1plus: todos los avisos se tratan como errores
make[2]: *** [CMakeFiles/VulkanHppGenerator.dir/build.make:76: CMakeFiles/VulkanHppGenerator.dir/VulkanHppGenerator.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/VulkanHppGenerator.dir/all] Error 2
~~~

greetings

--- a/VulkanHppGenerator.cpp
+++ b/VulkanHppGenerator.cpp
@@ -16545,7 +16545,7 @@ extern "C" __declspec( dllimport ) FARPROC __stdcall GetProcAddress( HINSTANCE h
 #if defined( CLANG_FORMAT_EXECUTABLE )
     std::cout << "VulkanHppGenerator: Formatting using ";
     std::string commandString = "\"" CLANG_FORMAT_EXECUTABLE "\" --version ";
-    std::system( commandString.c_str() );
+    (void)std::system( commandString.c_str() );
 #endif
 
     std::cout << "VulkanHppGenerator: Parsing " << filename << std::endl;

